### DISCUSSION TITLE
Transpile ts into js without bundeling

### DISCUSSION THREAD

**Discussion by DreamTexX** [denoland/deno#11302] [Q&A] [answered]
Hi, is it possible to generate the js output only without bundeling it into one file?

**Comment by ebebbington** [comment, marked as answer]
`Deno.emit`, or see https://github.com/ebebbington/crumpets

**Reply by ebebbington** [reply]
Also fyi when it’s no longer bundled into a single file, it’s also no longer bundling and (generally) becomes transpiling

**Reply by DreamTexX** [reply]
I know that it then is no bundeling anymore. Just didnt know how i should describe what i wanted to achive xd

Thanks Deno.emit is exactly the thing i was looking for!

**Reply by ebebbington** [reply]
No problem :) there's so many terms to call this stuff D:

**Reply by DreamTexX** [reply]
True... transpiling ts to js is also often just called compiling...
